tests/languages/nevod/keyword_feature.test

```diff
@@ -0,0 +1,13 @@
+@inside @outside @having
+@search @where

```

Comment:
Please add `@require`, `@namespace`, and `@pattern`.

Reply:
Test improved to check also for missed keywords.